app/Http/Transformers/CampaignTransformer.php

```diff
@@ -23,7 +23,8 @@ public function transform(Campaign $campaign)
             'is_open' => $campaign->isOpen(),
             'impact_doc' => $campaign->impact_doc,
             'cause' => $campaign->cause,
-            'pending_count' => $campaign->pending_count,
+            'accepted_count' => $campaign->accepted_count,
```

Comment:
Does this need `is_staff_user()` too?

Reply:
I don't think so, since this is the same as paging through the gallery & counting by hand.